### PR TITLE
feat: move delete all button and restyle side menu

### PR DESCRIPTION
**Pull request by tomwisecodes**
Moves delete all button and organises menu to match this ticket https://www.notion.so/oaknationalacademy/Gleap-clear-history-button-464dd50ad44c4bdb97f3857bbf9aa1c8

### CODE REVIEW

**Review by mantagen — COMMENTED** [review]
Looks good - few comments/suggestions

File: apps/nextjs/src/components/DialogControl/ContentOptions/ClearSingleChatFromChatHistory.tsx [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+import { trpc } from "@/utils/trpc";
+
+const ClearSingleChatFromChatHistory = ({
+  closeDialog,
+}: {
+  closeDialog: () => void;
+}) => {
+  const { dialogProps, setDialogProps } = useDialog();
+  const { mutate, isLoading } =
+    trpc.chat.appSessions.deleteChatById.useMutation({
```
[the remark this answers is not in the document]
Comment (tomwisecodes) [reply apps/nextjs/src/components/DialogControl/ContentOptions/ClearSingleChatFromChatHistory.tsx:24]: Lets do that has a follow up

File: apps/nextjs/src/components/AppComponents/Chat/sidebar-items.tsx [hunk 2]
```diff
@@ -11,22 +13,66 @@ interface SidebarItemsProps {
...
-            key={chat?.id}
-            exit={{
-              opacity: 0,
-              height: 0,
-            }}
-          >
+  const todayChats = parsedChats.filter((chat) => isToday(chat.updatedAt));
+  const lastWeekChats = parsedChats.filter(
+    (chat) =>
+      isThisWeek(chat.updatedAt, { weekStartsOn: 1 }) &&
```
[the remark this answers is not in the document]
Comment (tomwisecodes) [reply apps/nextjs/src/components/AppComponents/Chat/sidebar-items.tsx:27]: lets leave for now

File: apps/nextjs/src/components/AppComponents/Chat/Chat/types.ts [hunk 3]
```diff
@@ -5,4 +5,6 @@ export type DialogTypes =
   | "report-content"
   | "sensitive-moderation-user-comment"
   | "demo-interstitial"
-  | "demo-share-locked";
+  | "demo-share-locked"
+  | "clear-history"
+  | "clear-single-chat";
```
[the remark this answers is not in the document]
Comment (mantagen) [reply apps/nextjs/src/components/AppComponents/Chat/Chat/types.ts:10]: 🤦

**Review by mantagen — APPROVED** [review]
Code looks good!

**Review by mikeritson-oak — CHANGES_REQUESTED** [review]
Reviewed again and details within the Notion ticket for findings.

**Review by mikeritson-oak — APPROVED** [review]
These changes have been reviewed and confirmed. We have a couple of minor follow-up items to raise in separate tickets. 👍

**Pull request merged**